content/knowledge/sousakuten.md

```
@@ -0,0 +1,623 @@
+また、緊急避難時や体調不良者が出た場合に迅速な対応ができるよう、有事の際には前後どちらも一枚以上開けての対応ができるようにして、ドアの前には物を置かないでください。緊急時に開放できる仕組みになっているならば、換気時を含め前後どちらかの扉に限り、閉めっぱなしで構いません。 
+#### 換気について 
++ 立志部門 
+常時、出入り口は両方とも開放してください。また、常にエアコンの稼働をお願いします。 
++ 開拓・創作部門 
+公演間は必ず出入口と窓を開けて換気を行って下さい。換気の際は必ず、2台のサーキュレーターを稼働させてください。場所の指定は行いません。公演中はサーキュレーターを稼働させる必要はないので、各クラスで邪魔にならない場所にしま ってください。 
+サーキュレーターは、夏休み明けの配布を予定しています。 
+各クラスに配布されている消毒液は片づけずに、入口に設置してください。希望者が手指消毒を行えるようにしてください。 
+公演中は必ず、エアコンの稼働をお願いします。 
+#### その他 
+マニュアルに記載されている・いないに関わらず、情勢によっては内容の追加・変更を行う場合があります。その際は適宜連絡を行います、必ず従うようにしてください。大きな変更があった場合は個別に連絡を行うとは別に、創作展準備日までに完全版の配信を行います。
+### 団体支給品
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)